features/support/env.rb

```diff
@@ -1,5 +1,35 @@
 $LOAD_PATH.unshift(File.dirname(__FILE__) + '/../../lib')
-require 'aruba/cucumber'
+
+# Has to be the first file required so that all other files show coverage information
+require 'simplecov'
```

Comment:
Reviewing more of the support files and features, some of them are tagged `@in-process` and the only way to get in-process coverage in cucumber is for simplecov to be loaded in the cucumber process, so this require should stay for the @in-process tag.  If there wasn't that tag, it could have been removed.